core/lib/spree/core/controller_helpers/order.rb

```diff
@@ -79,7 +78,7 @@ def last_incomplete_order
         end
 
         def current_order_params
-          { currency: current_currency, guest_token: cookies.signed[:guest_token], store_id: current_store.id, user_id: try_spree_current_user.try(:id) }
+          {currency: current_currency, guest_token: cookies.signed[:guest_token], store_id: current_store.id, user_id: try_spree_current_user.try(:id)}
```

Comment:
Line is too long. [151/100]<br>Space inside { missing.<br>Space inside } missing.